zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -397,8 +400,65 @@ impl<'a, F: Field> EVMConstraintBuilder<'a, F> {
         self.query_cell_with_type(CellType::LookupByte)
     }
 
-    pub(crate) fn query_word_rlc<const N: usize>(&mut self) -> RandomLinearCombination<F, N> {
-        RandomLinearCombination::<F, N>::new(self.query_bytes(), self.challenges.evm_word())
+    // TODO remove me
+    // pub(crate) fn query_word_rlc<const N: usize>(&mut self) -> RandomLinearCombination<F, N> {
+    //     RandomLinearCombination::<F, N>::new(self.query_bytes(), self.challenges.evm_word())
+    // }
+
+    // default query_word is 2 limbs
+    pub(crate) fn query_word<const N: usize, const N2: usize>(
+        &mut self,
+        word_constrians: WordLimbs<Cell<F>, N2>,
+    ) -> Word<Cell<F>> {
+        let new_word = self._query_word();
+        self.require_equal(
+            "wordlimb equality",
+            word_constrians.expr().is_eq(&new_word.expr()),
```

Comment:
I noticed something here that generates confusion to me.
We have the `Expr` trait with the `expr()` method that returns `Expression<F>`
But `Word` implements `expr()` but instead of returning `Expression<F>` it returns `Word<Expression<F>>`.
So every time I see `.expr()` I think of a single expression, but here we have two` (the low and high).

So I would suggest finding a different name.  Maybe `word_expr()`?

Reply:
Good idea, renamed to `word_expr()` for clarification